docs/stdlib.md

```diff
@@ -248,6 +248,29 @@ kprobe:dummy {
 ktime_get_ns - ktime_get_boot_ns
 
 
+### elf_ino
```

Comment:
@Rtoax See the comments below. I don't think we need to expose these two builtins to users and instead just give them a different prefix.